mindjunkies/dashboard/tests/test_views.py

```
@@ -1,69 +1,146 @@
+
+    assert response.status_code == 200
+    assert "students" in response.context
+    assert len(response.context["students"]) == 1
+    assert response.context["students"][0] == enrollment.student
+
+
+@pytest.mark.django_db
+def test_remove_enrollment(client, teacher_user, course, enrollment):
+    client.force_login(teacher_user)
+
+    url = reverse("dashboard_enrollments_remove", kwargs={"course_slug": course.slug, "student_id": enrollment.student.uuid})
```

Ruff (0.8.2)

Line too long (125 > 119)

(E501)